src/main/k/in-progress/tests/collections/list/list-unify-any-elem/Makefile

```
@@ -1,19 +1,19 @@
-TOP := $(shell git rev-parse --show-toplevel)
+TOP != git rev-parse --show-toplevel
 include $(TOP)/include.mk
 
-KOMPILED := list-unify-1-kompiled
+KOMPILED := list-unify-any-elem-kompiled
 DEFINITION := $(KOMPILED)/definition.kore
 
-$(DEFINITION): list-unify-1.k $(KOMPILE_TARGETS)
-	$(KOMPILE) $(KOMPILE_OPTS) $< --syntax-module list-unify-1
+$(DEFINITION): list-unify-any-elem.k $(KOMPILE_TARGETS)
```

Please make this replacement in all the `Makefile`s:
```suggestion
$(DEFINITION): list-unify-any-elem.k $(K)
```